Source.cpp

```diff
@@ -0,0 +1,110 @@
+#include <iostream>
+#include <string>
+
+
+
+
+class Parser {
+public:
+	std::string mem1; std::string mem2; char oper;
+	std::string a;
+public:
+	void Parse(std::string a) {
+		int size = a.size();
+		int pos = a.find("-");
+		if (pos == -1) {
+			pos = a.find("+");
+		}
+		if (pos == -1) {
+			pos = a.find("*");
+		}
+		if (pos == -1) {
+			pos = a.find("/");
+		}
+		if (pos == -1) {
+			std::cout << "please use the operator" << std::endl;
+		}
+		else {
+			oper = a[pos];
+			for (int i = 0; i < pos; i++) {
+				mem1 += a[i];
```

Comment:
string has a [substr](https://en.cppreference.com/w/cpp/string/basic_string/substr) method, you could do the same more effectively with `mem1 = a.substr(0, pos);`